src/common/networks.ts

```diff
@@ -36,6 +37,10 @@ const jsonRpcProvider =
   () =>
     new providers.JsonRpcProvider(url);
 
+const getStabilityApiKey = (): string => {
+  return process.env.STABILITY_API_KEY ?? "";
+};
+
```

Comment:
can we use dotenv for this and use it directly for the rpcUrl

Reply:
This tool is supposed to be installed globally with `npm install —g`, so with `dotenv`, the execution of this CLI will depend on whether there is a `.env` in the execution path.

I read in the README that the project suggests using the `OA_PRIVATE_KEY` environment variable for some use cases where a Private Key is involved.

Maybe, as an alternative, we could document this env var in the README before we add constraints like the `.env` (like `OA_PRIVATE_KEY`)

Any thoughts?

Reply:
ok understand, forget what i just said, i think its fine for now. thanks